osu.Game/Screens/Play/BreakOverlay.cs

```diff
@@ -2,59 +2,67 @@
 // Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
 
 using System.Collections.Generic;
+using osu.Framework.Allocation;
+using osu.Framework.Configuration;
 using osu.Framework.Graphics;
 using osu.Framework.Graphics.Containers;
 using osu.Framework.Graphics.Shapes;
 using osu.Framework.Graphics.UserInterface;
 using osu.Game.Beatmaps.Timing;
+using osu.Game.Configuration;
 using osu.Game.Rulesets.Scoring;
 using osu.Game.Screens.Play.Break;
 
 namespace osu.Game.Screens.Play
 {
-    public class BreakOverlay : Container
-    {
-        private const double fade_duration = BreakPeriod.MIN_BREAK_DURATION / 2;
-        private const float remaining_time_container_max_size = 0.3f;
-        private const int vertical_margin = 25;
+  public class BreakOverlay : Container
+  {
+    public BindableFloat BackgroundOpacityOffset;
 
-        private List<BreakPeriod> breaks;
+    private const double fade_duration = BreakPeriod.MIN_BREAK_DURATION / 2;
+    private const float remaining_time_container_max_size = 0.3f;
+    private const int vertical_margin = 25;
 
-        private readonly Container fadeContainer;
+    private List<BreakPeriod> breaks;
 
-        public List<BreakPeriod> Breaks
-        {
-            get => breaks;
-            set
-            {
-                breaks = value;
-                initializeBreaks();
-            }
-        }
+    private Bindable<bool> lightenDuringBreaks;
+    private readonly BindableBool breakLighten = new BindableBool();
```

Comment:
Two Bindables are unnecessary. You can get rid of one.

Reply:
I'm not sure If I understand. `lightenDuringBreaks` is for the setting and `breakLighten` is set to true when there is a break. How would I get rid of one?